coreos-base/oem-ec2-compat/oem-ec2-compat-0.1.2-r2.ebuild

```diff
@@ -63,4 +63,6 @@ src_install() {
 	if use ec2 ; then
 		newins "${FILESDIR}/base/base-ec2.ign" base.ign
 	fi
+
+	use openstack && newins "${FILESDIR}/base/openstack.ign" base.ign
```

Comment:
Does the return code of `src_install` matter or can it be non-0?

Reply:
`newins` calls `die` if there is an error during the installation of the file: this operation should succeed or die.

Reply:
I think the question is whether the return value of `src_install` has any meaning. This function's return value will the return value of `use openstack && newins "…"`, which will be 1 for non-openstack images (because `use openstack` will return 1).

Reply:
I think I'd recommend rewriting this as `if use openstack; then newins "…"; fi` for consistency.

Reply:
Yeah I tend to follow this documentation: https://devmanual.gentoo.org/ebuild-writing/use-conditional-code/index.html
> For single-statement conditions, the use foo && blah (or use foo || blah for negatives) form is often more readable.

But yes, in this case it's wrong to return `use openstack && newins "…"` as it will be 1 for non-openstack image (AWS). Thanks!